app/models/conference.rb

```diff
@@ -328,9 +328,10 @@ def event_distribution
   # ====Returns
   # * +hash+ -> hash
   def scheduled_event_distribution
-    confirmed_events = program.events.where(state: 'confirmed')
-    scheduled_value =  { 'value' => confirmed_events.where.not(start_time: nil).count, 'color' => 'green' }
-    unscheduled_value =  { 'value' => confirmed_events.where(start_time: nil).count, 'color' => 'red' }
+    confirmed_scheduled_events = Event.scheduled.select{|event| event.program == program}
```

Comment:
@Ana06 Yes it worked :wink:.But since our Program model also has a method named scheduled so I just changed the name of our scope.